modules/updateTime.js

```diff
@@ -0,0 +1,13 @@
+import { DateTime } from './luxon.js';
+
+const updateTime = () => {
+  const now = DateTime.now();
+  const options = { month: 'short', day: 'numeric', year: 'numeric' };
+  const date = now.toLocaleString(options);
+  const time = now.toLocaleString(DateTime.TIME_WITH_SECONDS);
+  document.getElementById('dateTime').innerHTML = `${date} , ${time}`;
+};
+
+setInterval(updateTime, 1000);
+
+export default updateTime;
```

Comment:
- [ ] Kindly add this on the right side of the app just like the given wireframe:

Yours:
![image](https://github.com/SAZZAD1Q2/Awesome-Book/assets/62705947/1564910b-0a5d-4396-b15c-77ba1e273e45)

Design:
![image](https://github.com/SAZZAD1Q2/Awesome-Book/assets/62705947/416fd752-2e2b-4485-ab60-aca314bea973)

Reply:
Hello
Thank you so much for the suggestion. I actually forgot to do so. Now I am setting the dateTime right side to submit again.
Thank you once again.
Sazzad